src/main/scala/org/broadinstitute/dsde/rawls/webservice/MethodConfigApiService.scala

```diff
@@ -184,28 +184,27 @@ trait MethodConfigApiService extends HttpService with PerRequestCreator {
     }
   }
 
-  @Path("/copy")
+  @Path("/methodconfigs/copy")
```

Comment:
unfortunately this does not produce the desired swagger description but I can't figure out what would